tests/end-to-end/external-solution-integration/pkg/testsuite/deploy_function.go

```diff
@@ -32,8 +32,17 @@ function rejected(result) {
   console.log('Rejected:');	
   console.log(JSON.stringify(result, null, 2));	
 }
-function sendReq(url, resolve, reject) {	
-    request.post(url, { json: true }, (error, response, body) => {	
+function sendReq(event, url, resolve, reject) {
```

Comment:
please reindent this js snippet, it makes it easier to read

Reply:
Fixed